README.md

```diff
@@ -12,15 +12,20 @@ A Quick-start Guide for people that want to **D**o **W**hat **Y**ou **L**ove!
 The world is not perfect.  
 We have all worked somewhere we _didn't **love**_. <br />
 We have _all_ felt the _frustration_ of using products that have flaws
-(and been unable to fix them).   
-**The aim of _dwyl_ is to address *both* of these issues _simultaneously_.**
-
-If you'd like to know more, _please **read our [manifesto](/manifesto.md)**_.  
+that _we_ were unable to fix ourselves.   
+We have all experienced being in an environment that
```

Comment:
I added this sentence because I felt it brought this paragraph more 'together' but ultimately it will need to be rewritten as I've mentioned. We can remove it if it doesn't make sense.